### DISCUSSION TITLE
Sponsor

### DISCUSSION THREAD

**Discussion by Ernst79** [custom-components/ble_monitor#473] [General]
@Thrilleratplay , @Magalex2x14 I want to add both of you to the sponsor button of BLE monitor, if you want. If you want to be added, I guess you will have to enable this first and fill in your bank account somewhere in your GitHub account. 

Don’t expect to become rich, :smile so far I have received 10 Euro I think, just see it as an appreciation of the users. Let me know if you want to be added!

**Comment by Thrilleratplay** [comment]
@Ernst79 I appreciate the offer and, while tempted by such wealth, I do what I can to give whenever possible to back to the open source community on a voluntary basis.

**Comment by Magalex2x14** [comment]
I don't need that either. Moreover, for a long time I have not been able to devote as much time to this project as I would like...
